spec/routes/api/cli/golden-files/lb eu-central-h1_test-lb update bad 1234 54321 _up2 vmdzyppz6j166jh5e9t2dwrfas.txt

```diff
@@ -0,0 +1,2 @@
+! Unexpected response status: 400
+Details: algorithm is not in range or set: ["round_robin", "hash_based"]
```

Comment:
Relating to suggestions on the error above. When there is just one line, it seems fine to not have a newline, but when there were two I guess it felt somewhat different. I think with other things the pattern we leaned toward was newline even if there was only one line (ie for the nested serializations).

In a sense, if this is a one item unordered list, it should probably be newline dileneated if that is what we are doing elsewhere.

Reply:
The part following `Details: ` is from the `error.message`, and is always a single line.  There could be zero or more lines after that, one for each entry in `error.details`.  Those provide details for the message, so if you were going to indent the message, I think the proper way would be this:

```
Details:
  Validation failed for following fields: vm_id, foo
    vm_id: VM is already attached to a load balancer
    foo: some other error message
```

I don't think the additional nesting improves the output, but if you do, please let me know and I'll make that change.